modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/MouseBasePhenotypeAnnotation.pm

```diff
@@ -0,0 +1,452 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+
+=head1 MouseBasePhenotypeAnnotation
+
+General mouse phenotype import methods.
+
+=cut
+
+package Bio::EnsEMBL::Variation::Pipeline::PhenotypeAnnotation::MouseBasePhenotypeAnnotation;
+
+use strict;
+use warnings;
+use HTTP::Tiny;
+use JSON;
+
+use base ('Bio::EnsEMBL::Variation::Pipeline::PhenotypeAnnotation::BasePhenotypeAnnotation');
+
+my ($logFH, $errFH);
+
+sub set_logFH {
```

Comment:
Do we need to redefine the methods `set_logFH` and `set_errFH` if they are already defined in `BasePhenotypeAnnotation` ?